### PR TITLE
Update examples after mpes refactor

### PR DESCRIPTION
**Pull request by lukaspie**

### CODE REVIEW

File: pynxtools_xps/config/config_txt_vamas_export.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
         "resolution":"@xps_token:data/step_size",
         "resolution/@units":"@eln"
       },
-      "source_TYPE[source_probe]":{
+      "source_TYPE[source_xray]":{
```
Comment (domna) [inline pynxtools_xps/config/config_txt_vamas_export.json:39]:
```suggestion
      "sourceTYPE[source_xray]":{
```

File: pynxtools_xps/config/config_txt_vamas_export.json [hunk 2]
```diff
@@ -45,9 +45,9 @@
           "identifier":"@eln",
           "model":"@eln"
         },
-        "associated_beam":"@link:/entry/instrument/beam_probe"
+        "associated_beam":"/entry/instrument/beam_xray"
       },
-      "beam_TYPE[beam_probe]":{
+      "beam_TYPE[beam_xray]":{
```
Comment (domna) [inline pynxtools_xps/config/config_txt_vamas_export.json:50]:
```suggestion
      "beamTYPE[beam_xray]":{
```

File: pynxtools_xps/config/config_vms.json [hunk 3]
```diff
@@ -45,11 +45,20 @@
...
+       "associated_beam":"/entry/instrument/beam_xray",
+       "depends_on": "/entry/instrument/source_xray/transformations/source_analyzer_angle",
+       "TRANSFORMATIONS[transformations]": {
+         "AXISNAME[source_analyzer_angle]":"@xps_token:source/source_analyzer_angle",
+         "AXISNAME[source_analyzer_angle]/@transformation_type": "rotation",
+         "AXISNAME[source_analyzer_angle]/@units":"degree",
+         "AXISNAME[source_analyzer_angle]/@vector": [
+           0,
+           0,
+           1
```
Comment (domna) [inline pynxtools_xps/config/config_vms.json:57]: This should ideally also contain a `depends_on` (can be `.` if this refers to the base coordinate system)

**Review by domna — CHANGES_REQUESTED** [review]

File: pynxtools_xps/config/config_vms.json (hunk 3 above)
Comment (lukaspie) [reply pynxtools_xps/config/config_vms.json:58]: Makes sense, I added it in all config files that contain transformations.

**Comment by lukaspie** [comment]
@domna thanks for the review, I addressed all your comments.

**Review by domna — APPROVED** [review]

**Pull request merged**